code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -432,16 +432,16 @@
 
 	//They're injured enough for it!
 	var/list/treat_me_for = list()
-	if(C.getBruteLoss() >= heal_threshold)
+	if((C.getBruteLoss() >= heal_threshold) && C.getBruteLoss() != 0)
 		treat_me_for += BRUTE
 
-	if(C.getOxyLoss() >= (5 + heal_threshold))
+	if((C.getOxyLoss() >= heal_threshold) && C.getOxyLoss() != 0)
```

Comment:
The +5 has been truncated.

Reply:
Ack, good catch. That's what I get for copy pasting.